core/src/main/java/nu/yona/server/goals/service/ActivityCategoryService.java

```diff
@@ -225,9 +225,9 @@ private String getName(ActivityCategory entity)
 
 	/**
 	 * This method calls flush on the activity category repository. It shouldn't be necessary to do this, but the implicit flush
-	 * (done just before JPA sends a query to the database) causes a
```

Comment:
This seems a slight difference in line length. That might be the cause of the accidental formatting difference. Please keep flagging them.